### PR TITLE
chore(dependency): update jest

### PR DESCRIPTION
**Pull request by dbanksdesign**
*Issue #, if available:*

*Description of changes:* This will fail, but creating the PR to see tests fail in Travis

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by crunchbar** [comment]
Pick what you need and try adding the below to your `package.json`

```JSON
{
  "jest": {
    "moduleFileExtensions": [
      "web.js",
      "js",
      "web.ts",
      "ts",
      "web.tsx",
      "tsx",
      "json",
      "json5",
      "web.jsx",
      "jsx",
      "node"
    ]
  }
}
```

**Comment by dbanksdesign** [comment]
Thank you @crunchbar, but I am still getting the issue. No new errors, it didn't seem to have any effect.

**Comment by dbanksdesign** [comment]
Ok I think I figured it out. It had to do with my old .babelrc file. I updated to `@babel/preset-env`

**Comment by chazzmoney** [comment]
LGTM! :shipit:

File: package.json [hunk 1]
```diff
@@ -65,7 +65,10 @@
     "transform": {
       "^.+\\.json5?$": "json5-jest",
       "^.+\\.jsx?$": "babel-jest"
-    }
+    },
+    "transformIgnorePatterns": [
+      "^.+\\.json?$"
```
Comment (crunchbar) [inline package.json]: Did you mean `"^.+\\.json$"` without the question mark?

The question mark makes the `n` optional.

File: package.json [hunk 2]
```diff
@@ -65,7 +65,10 @@
     "transform": {
       "^.+\\.json5?$": "json5-jest",
```
Comment (crunchbar) [inline package.json]: If you just want to target `json5` here, you can remove the question mark `?`.  By doing so, you wouldn't need to specify the `transformIgnorePatterns` below.

Comment (dbanksdesign) [reply package.json]: Made that change, thank you!

File: package.json (hunk 1 above)
Comment (dbanksdesign) [reply package.json]: Good catch! I made the change below and removed transformIgnorePatterns completely and it works! 👍

**Pull request merged**